programs/steward/src/state/accounts.rs

```diff
@@ -44,26 +63,6 @@ impl Config {
     }
 }
 
-// PDA that is used to sign instructions for the stake pool.
-// The pool's "staker" account needs to be assigned to this address,
-// and it has authority over adding validators, removing validators, and delegating stake.
-#[account]
-pub struct Staker {
-    pub bump: u8,
-}
-impl Staker {
-    pub const SIZE: usize = 8 + size_of::<Self>();
-    pub const SEED: &'static [u8] = b"staker";
-
-    pub fn get_address(config: &Pubkey) -> Pubkey {
-        let (pubkey, _) =
-            Pubkey::find_program_address(&[Self::SEED, config.as_ref()], &crate::id());
-        pubkey
-    }
-}
-
-// static_assertions::const_assert_eq!(StewardStateAccount::SIZE, 162584);
```

Comment:
let's keep this - still want to uncomment this once State size is finalized. Here to ensure that the account size doesn't change in future PRs after it goes live